kuka_kr150_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr150_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR150-2 (and variants).
```

Comment:
I believe the KR150-2 is itself a variant, is it not? If so, this should probably read:

> ROS-Industrial support for the KUKA KR 150 (and variants).